velox/exec/ExchangeQueue.h

```diff
@@ -26,7 +26,8 @@ class SerializedPage {
   // Construct from IOBuf chain.
   explicit SerializedPage(
       std::unique_ptr<folly::IOBuf> iobuf,
-      std::function<void(folly::IOBuf&)> onDestructionCb = nullptr);
+      std::function<void(folly::IOBuf&)> onDestructionCb = nullptr,
+      std::optional<int64_t> rows = std::nullopt);
```

Comment:
naming: numRows for consistency

Please, add @param comment to document the new parameter

Reply:
Naming done, but it seems that the @param annotation is not used in the velox project.